developer-docs-site/docs/nodes/indexer-fullnode.md

```diff
@@ -80,6 +80,12 @@ For an Aptos indexer fullnode, install these packages:
         emit_every: 500
     ```
 
+:::tip Bootstap the fullnode
+Instead of syncing your indexer fullnode from genesis, which may take a long period of time, you can choose to bootstrap your fullnode using backup data before starting it. To do so, follow the instructions to [restore from a backup](../nodes/full-node/aptos-db-restore.md).
+
+Note: indexers cannot be bootstrapped using [a snapshot](../nodes/full-node/bootstrap-fullnode.md) or [fast sync](../guides/state-sync.md#fast-syncing), as these are currently unsupported.
```

Comment:
I think the phrasing here of "currently unsupported" isn't quite right as I don't see how it could ever be supported...